DVSRegister/Views/Cookie/CookiePage.cshtml

```diff
@@ -8,7 +8,7 @@
 @using (Html.BeginForm("CookieSettings_Post", "Cookie", FormMethod.Post))
 {
     <div class="govuk-width-container">
-        @* @Html.ActionLink("Back", "ConfirmityExpiryDate", "Cab", null, new { @class = "govuk-back-link" }) *@
+        @Html.ActionLink("Back", "StartPage", "PreRegistration", null, new { @class = "govuk-back-link" })
```

Comment:
@poojadsit  We wont be using Preregistration any more, can you make it cab landing page?